packages/insomnia-prettify/src/json.ts

```diff
@@ -35,7 +40,7 @@ module.exports.prettify = function(json, indentChars = '\t', replaceUnicode = tr
   // we need to convert all escaped unicode characters to proper unicode characters.
   if (replaceUnicode) {
     try {
-      json = _convertUnicode(json);
+      json = convertUnicode(json);
```

Comment:
We currently have a convention in this codebase where we prefix private functions/variables with `-`.
While we can discuss whether this is still needed now that we use TS and change that convention, I don't think this should be done during the conversion, but rather separately in the future, partially because it's a team decision that needs to be made, and partially because renames like these requires a lot more involved review to ensure that no instances of a variable were/there's no accidental shadowing/etc, so please revert this.